conf/vmaas-sync.env

```diff
@@ -8,3 +8,5 @@ ENABLE_REPO_BASED_RE_EVALUATION=YES
 VMAAS_REQUEST_RETRIES=10
 RE_EVALUATION_KAFKA_BATCH_SIZE=10000
 RE_EVALUATION_KAFKA_BATCHES=10
+DEFAULT_PAGE_SIZE=5000
+DEFAULT_PAGE=1
```

Comment:
I would probably get rid of this environment variable, I don't see any reason why we would need an env variable for first page.

Reply:
I agree. Personally, I'd keep DEFAULT_PAGE_SIZE in env variable. DEFAULT_PAGE is not neccessary in env variable. I don't find why we would start fetching data form vmaas with other default page than first page.